strength_percentiles.py

```diff
@@ -94,40 +106,82 @@ def get_data_from_table(row, column):
             return data
     return data
 
-def categorize_and_store_result(container, result, categories):
-    """The input is an iterable of categories containing a gender, professional
-    status, an equipment status (raw or equipped).  This function stores the
-    meet results in the appropriate subsections of the container."""
-    gender, professional_status, equipment = categories
-    container['all'].append(result)
-    if gender == 'Female':
-        container['women'].append(result)
-    else:
-        container['men'].append(result)
+def populate_database(webpage, database):
+    """
+    Creates SQLite database with specified name.  Creates table inside the
+    database and populates it with results of powerlifting meets.  If the
+    created table has the same name as an already existing table, the already
+    existed table is deleted.
 
-    if professional_status == 'AM':
-        container['amateurs'].append(result)
-    else:
-        container['professionals'].append(result)
+    Paramaters:
+    -----------
+    database: string
+        String specifying the name of the database in which to store meet
+        results.
+    webpage: string
+        String specifying webpage from which to scrape meet results.
 
-    if equipment.startswith('Raw'):
-        container['raw'].append(result)
-    else:
-        container['equipped'].append(result)
-
-def test_scrape_webpage_and_store_meet_results(container):
-    for category in container:
-        print category, "~"*15
-        for result in container[category]:
-            print "\t", result
-        print "\n"*5
-    print "Number of lifters: ", len(container['all'])
-    print "Number of men: ", len(container['men'])
-    print "Number of women: ", len(container['women'])
-    print "Number of amateurs: ", len(container['amateurs'])
-    print "Number of professionals: ", len(container['professionals'])
-    print "Number of raw lifters: ", len(container['raw'])
-    print "Number of equipped lifters: ", len(container['equipped'])
+    """
+    table_deletion_string = "drop table if exists meet_results"
+
+    create_table_string = """CREATE TABLE meet_results (
+    lifter_id INTEGER PRIMARY KEY, gender TEXT,
+    professional_status TEXT, equipment TEXT, squat REAL,
+    bench REAL, deadlift REAL, total REAL)
+    """
+
+    # IMPORTANT NOTE: Instead of using the SQLite's parameterized query
```

Comment:
This comment on explaining why you dont manually format a string query is unnec.